js/chat_app/src/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable jsx-quotes */
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { setStatus } from './redux/slicies/testSlise';
 
 function App() {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.test);
```

Comment:
I think that if you want to create example - please use something that can make sense

For example, you gonna have authorization - so create "auth" slice for redux-toolkit and leave a simple example where you will be dispatching any action.